backend/sources/google/sender/src/main/java/co/airy/core/sources/google/GoogleConfig.java

```diff
@@ -0,0 +1,29 @@
+package co.airy.core.sources.google;
+
+import co.airy.core.sources.google.model.GoogleServiceAccount;
+import co.airy.log.AiryLoggerFactory;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class GoogleConfig {
+    private static final Logger log = AiryLoggerFactory.getLogger(GoogleConfig.class);
+    final ObjectMapper objectMapper;
+    public GoogleConfig(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    @Bean
+    public GoogleServiceAccount googleServiceAccount(@Value("${google.auth.sa}") String googleServiceAccountFile) {
```

Comment:
For the google sender I would take the approach we took in the tedi automation app. Instead of accessing the google's `channel→token` users provide a single line version of the Service Account file. This means:

- `token` becomes optional for Google channels
- We don't need a channel's table in the sender app

As long as we do not have product (user) input for this, I believe that this approach fits the Google authentication flow better.